source/dataman/CacheMan.cpp

```diff
@@ -106,75 +101,73 @@ void CacheMan::push()
 void CacheItem::push()
 {
     size_t varbytes = m_jmsg["varbytes"].get<size_t>();
-    std::vector<char> buff(varbytes);
-    clean(buff, m_clean_mode);
-    m_cache.push(buff);
-}
-
-void CacheItem::clean(std::string a_mode) { clean(m_cache.front(), a_mode); }
-void CacheMan::clean(std::string a_mode)
-{
-    for (auto i : m_cache)
-    {
-        for (auto j : i.second)
-        {
-            j.second.clean(a_mode);
-        }
-    }
+    m_cache.push(std::vector<char>(varbytes));
```

Comment:
Use `emplace(varbytes)` instead of `push(std::vector<char>(varbytes))` here to create the buffer in-place on the queue instead of copying it.  You could also explicitly move the value with `push(std::move(std::vector<char>(varbytes)))` but really jsut calling `emplace` is much cleaner.